src/backend/distributed/metadata/metadata_sync.c

```diff
@@ -679,21 +679,24 @@ MetadataCreateCommands(void)
 	/* after all tables are created, create the metadata */
 	foreach_ptr(cacheEntry, propagatedTableList)
 	{
-		Oid clusteredTableId = cacheEntry->relationId;
+		Oid relationId = cacheEntry->relationId;
```

Comment:
Adding a new node fails:
```SQL
select citus_add_node('localhost', 9706);
ERROR:  operation is not allowed on this node
HINT:  Connect to the coordinator and run it again.
CONTEXT:  while executing command on localhost:9706
Time: 95.080 ms
```

It seems we somewhere miss `set citus.enable_ddl_propagation to off;` and the following fails:

```SQL
CREATE SERVER IF NOT EXISTS foreign_server TYPE 'test_type' VERSION 'v1' FOREIGN DATA WRAPPER postgres_fdw OPTIONS (host 'localhost', port '5432', dbname 'otherdb')
```

Reply:
We should make sure that `add_node` propagates types used on the foreign table (or owner / schema etc) to the new node

Reply:
Removed server and extension generation commands from `GetPreLoadTableCreationCommands` as now we send server creation commands to the workers by default. That fixed this issue. We were sending CREATE SERVER commands twice before that.
Yet I'm not sure whether this is a good solution or not. Need to test more

Reply:
can we add a test? maybe to another file where we already add node?